modules/apps/wiki/wiki-service/service.xml

```diff
@@ -312,6 +312,7 @@
 	<exceptions>
 		<exception>DuplicateNodeName</exception>
 		<exception>DuplicatePage</exception>
+		<exception>DuplicateWikiNodeExternalReferenceCode</exception>
```

Comment:
Does this follow the same convention as MBMessage?

Reply:
Yes, the same convention that MBMessage and DLFileEntry